### PR TITLE
[MDEP-939] Lock down classifier in dependency:sources goal

### PR DESCRIPTION
**Pull request by michael-o**
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MDEP) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[MDEP-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MDEP-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [ ] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [ ] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

File: src/it/projects/sources/test.properties [hunk 1]
```diff
@@ -16,3 +16,4 @@
 # under the License.
 
 outputFile = target/resolved.txt
+classifier = foo
```
Comment (michael-o) [inline src/it/projects/sources/test.properties:19]: This may require a comment that it has no influence anymore...

**Comment by slawekjaranowski** [comment]
Looks ok - even more `classifier` parameter disappears from plugin documentation

**Review by slawekjaranowski — APPROVED** [review]

**Comment by michael-o** [comment]
> Looks ok - even more `classifier` parameter disappears from plugin documentation

This is a nice bonus or a bug in the Plugin Tools.

**Comment by slawekjaranowski** [comment]
> > Looks ok - even more `classifier` parameter disappears from plugin documentation
> 
> This is a nice bonus or a bug in the Plugin Tools.

works as expected - documentation was taken from the final class - as we have read-only property is hidden

**Comment by michael-o** [comment]
> > > Looks ok - even more `classifier` parameter disappears from plugin documentation
> > 
> > 
> > This is a nice bonus or a bug in the Plugin Tools.
> 
> works as expected - documentation was taken from the final class - as we have read-only property is hidden

Perfect, thanks.

File: src/it/projects/sources/test.properties (hunk 1 above)
Comment (michael-o) [reply src/it/projects/sources/test.properties:19]: Will do

Comment (slawekjaranowski) [reply src/it/projects/sources/test.properties:19]: this test can be useful ... even we have a read only property user still can override it

Comment (slawekjaranowski) [reply src/it/projects/sources/test.properties:19]: and there are no assertions in this test

Comment (michael-o) [reply src/it/projects/sources/test.properties:19]: I guess at least the build should not fail, but need to try...

**Comment by michael-o** [comment]
FTR: All resolve ITs missing verifiers, this needs to be addressed as a whole.

**Pull request merged**